media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+jQuery(document).ready(function ($)
+{
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $().val(),
```

Comment:
@roland-d , any specific way to pass the id here? I have comparatively limited knowledge in js